packages/mobile/android/build.gradle

```diff
@@ -4,8 +4,8 @@ buildscript {
     ext {
         buildToolsVersion = "29.0.2"
         minSdkVersion = 16
-        compileSdkVersion = 28
-        targetSdkVersion = 28
+        compileSdkVersion = 29
```

Comment:
why bump to 29?

Reply:
This was necessary for react-native-keychain to work properly